### PR TITLE
[doc] Update how-to-build-runtime.md

### PR DESCRIPTION
**Pull request by ragmani**
This commit updates how-to-build-runtime.md.
  - Add installing libglib2.0-dev for x64 build

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

**Comment by ragmani** [comment]
#9386 requires the installation of `libglib2.0-dev` package to build onert.

File: docs/howto/how-to-build-runtime.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ If you are going to build this project, the following modules must be installed
 In the Ubuntu, you can easily install it with the following command.
 
 ```
-$ sudo apt-get install cmake libboost-all-dev
+$ sudo apt-get install cmake libboost-all-dev libglib2.0-dev
```
Comment (hseok-oh) [inline docs/howto/how-to-build-runtime.md:15]: We need to update above list (line 9~10) amd below `apt` command (line 23~38)

Comment (hseok-oh) [reply docs/howto/how-to-build-runtime.md:15]: If we install simulator, we may use `npud` on x64.

cc/ @jyoungyun

**Comment by ragmani** [comment]
> IMO, it's better to make separate part for system service because it is optional depend on platform yet.

OK, I will close this PR. And I have no plan to write a separate part for system service because I don't know about it well.

https://github.com/Samsung/ONE/blob/debeb771d15c6d3efe1f5449ffc0164457919e73/infra/nnfw/cmake/CfgOptionFlags.cmake#L34
https://github.com/Samsung/ONE/blob/debeb771d15c6d3efe1f5449ffc0164457919e73/runtime/service/npud/CMakeLists.txt#L1-L7

By the way, `npud` is built as default now. It looks like `npud` is not optional.

**Comment by glistening** [comment]
> By the way, npud is built as default now. It looks like npud is not optional.

I think it would be good default off. `npd` does nothing with `onert` at this moment.

Currently, everyone who want to use `onert` have to install `glib` and build `npud`, which is optional at this moment.

cc @Samsung/one_onert, @jyoungyun

**Comment by jyoungyun** [comment]
I agree with this point. Currently, npu daemon is not ready and not activated yet. So we can change it to false by default.

**Pull request closed without merging**